source/extensions/filters/http/cache/http_cache.cc

```diff
@@ -159,6 +175,94 @@ bool adjustByteRangeSet(std::vector<AdjustedByteRange>& response_ranges,
   }
   return true;
 }
+
+std::vector<RawByteRange> RangeRequests::parseRanges(const Http::RequestHeaderMap& request_headers,
+                                                     int byte_range_parse_limit) {
+  // Makes sure we have a GET request, as Range headers are only valid with this type of request.
+  const absl::string_view method = request_headers.getMethodValue();
+  ASSERT(method == Http::Headers::get().MethodValues.Get);
+
+  // Multiple instances of range headers are invalid.
+  // https://tools.ietf.org/html/rfc7230#section-3.2.2
+  std::vector<absl::string_view> range_headers;
+  Http::HeaderUtility::getAllOfHeader(request_headers, Http::Headers::get().Range.get(),
+                                      range_headers);
+
+  absl::string_view range;
+  if (range_headers.size() == 1) {
+    range = range_headers.front();
+  } else {
+    ENVOY_LOG(debug, "Multiple range headers provided in request. Ignoring all range headers.");
```

Comment:
Does this get logged for GET requests with no range headers?

Reply:
I believe it did, I fixed it to only show for when number of headers > 1.